### ISSUE TITLE
Feature Request: Instructions for zelda space saving with upgraded RAM

### ISSUE THREAD

**Issue by retrohead** [BrianPugh/game-and-watch-patch#44]
Could you provide the offsets if we want to use the zelda space saving mod with a 16MB and a 64MB chip as these are most common upgrades?

https://github.com/BrianPugh/game-and-watch-patch/blob/main/docs/zelda.md

Thanks

**Comment by BrianPugh** [comment]
The offsets mentioned there point to an "island" of free storage (1802240 bytes) in the middle of the firmware by removing the Link's Awakening ROMs and the sleeping images. Because of this, it doesn't benefit other flash chip sizes. Another unfortunate thing is that there's an additional 797,552 bytes "island" of free storage at offset 0x325490, but we need contiguous regions (or need to modify retro-go's linker script to accept mutliple noncontiguous regions to place roms).

For the mario rom, I hand-relocated all the bits in the external storage to free up the space. The Zelda ROM is a bit more difficult, and honestly i've put less effort into it.

tl;dr, there's currently no space saving techniques for the zelda ROM that would add more free space for larger chips. So, for now, you're stuck with the zelda rom taking up the first 4MB of your ext flash.

**Comment by retrohead** [comment]
Thanks for the explanation. That explains perfectly why I couldn't make sense of the sizes.
